### ISSUE TITLE
[Feature Request]: Support for Laravel Lift Library

### ISSUE THREAD

**Issue by abishekrsrikaanth** [laravel-idea/plugin#820]
### Feature Description

Can you provide support for [Laravel Lift Library](https://github.com/WendellAdriel/laravel-lift). Most of the stuff is transformed into variable and class level attributes and it currently doesn't have support for IDE with attributes. Would be great if this plugin can help with that.

**Comment by adelf** [comment]
What kind of support do you mean?

**Comment by abishekrsrikaanth** [comment]
for example, [relationships](https://wendell-adriel.gitbook.io/laravel-lift/attributes/relationships) are defined as attributes at the class level

```php
#[BelongsTo(User::class)]
final class Post extends Model
{
    use Lift;
    // ...
}
```
right now PHPStorm/PHPstan shows an error saying that `Relation 'user' is not found in model 'Post'`. If the laravel idea generator is able to create appropriate methods in the generator file, then this wouldn't happen. and we kind of loose a lot of intellisense. For example

```php
$post = Post::with('user')
```
currently without Laravel Lift, when use open the brackets for `with(` it automatically shows a list of relations that can be loaded. Does not happen anymore when we use Lift to define relationships. Let me know if it makes sense.

**Comment by WendellAdriel** [comment]
If I can help with anything here just let me know! I’ll be happy to help! 💪

**Comment by adelf** [comment]
@WendellAdriel thanks! I've just created a PR with some completions. https://github.com/WendellAdriel/laravel-lift/pull/42

@abishekrsrikaanth I'll try to support this package.

**Comment by abishekrsrikaanth** [comment]
> @WendellAdriel thanks! I've just created a PR with some completions. [WendellAdriel/laravel-lift#42](https://github.com/WendellAdriel/laravel-lift/pull/42)
> 
> @abishekrsrikaanth I'll try to support this package.

Thank you. Looking forward to it

**Comment by adelf** [comment]
Implemented in 7.4
